Library/Homebrew/cmd/update-report.rb

```diff
@@ -560,3 +555,13 @@ def installed?(formula)
     (HOMEBREW_CELLAR/formula.split("/").last).directory?
   end
 end
+
+class Docs
```

Comment:
Per the comment on the completions / manpages class, it's likely not a class - just a method. Which likely belongs here as the only user of this. Or should this be extended to allow taps to install to docs?